web-api/src/storage/elastic.rs

```diff
@@ -247,7 +247,8 @@ pub struct Document {
     pub snippet: String,
     pub properties: DocumentProperties,
     pub embedding: Embedding,
-    pub category: Option<String>,
+    #[serde(alias = "category")] // deprecated
```

Comment:
It is not in the spec so I think is ok to not support it. Is there some case I'm not aware of?

Reply:
i guess it only matters if we actually had data stored in elastic. i'll remove it.